src/cli/commands/status.rs

```diff
@@ -28,7 +28,43 @@ pub enum StatusCommand {
 impl Status {
     pub async fn execute(self, client: CascadeApiClient) -> Result<(), String> {
         match self.command {
-            Some(_) => todo!(),
+            Some(StatusCommand::Keys) => {
+                let response: KeyStatusResult = client
+                    .get("/status/keys")
+                    .send()
+                    .and_then(|r| r.json())
+                    .await
+                    .map_err(format_http_error)?;
+
+                println!("First to expire (max 5):");
+                if response.expirations.is_empty() {
+                    println!("   No keys will expire");
+                }
+
+                for e in response.expirations.iter().take(5) {
```

Comment:
It's a bit wasteful to return all but only use 5, should this maybe be handled on the server side?